pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -129,7 +130,7 @@ type PersistentVolumeController struct {
 	claims  cache.Store
 
 	// Map of scheduled/running operations.
-	runningOperations goroutinemap.GoRoutineMap
+	runningOperations goroutinemap.GoroutineMap
```

Comment:
@saad-ali @justinclayton Let's do the renaming in a separate PR. For 1.3 it doesn't make sense. And it makes the review here much more complicated to see the minimal changes to goRoutineMap.